Sources/StructuredQueriesSQLiteCore/Internal/Deprecations.swift

```diff
@@ -1,11 +1,123 @@
 import Foundation
 import StructuredQueriesCore
 
+// NB: Deprecated after 0.22.2:
+
+extension Table {
+  @available(
+    *,
+    deprecated,
+    message: "Prefer 'createTemporaryTrigger(after: .update(touch:))', instead"
+  )
```

Comment:
Worth soft-deprecating, or is the surface minimal enough to push folks to the new APIs sooner?